bigtop_toolchain/manifests/packages.pp

```diff
@@ -401,7 +401,7 @@
   # download python 2.7.14 for openEuler docker slaves
   # and RHEL9 based distros
   if $operatingsystem == 'openEuler'
-     or (/redhat|centos|rocky/ in $operatingsystem and Integer($operatingsystemmajrelease) >= 9) {
+     or ($osfamily == 'RedHat' and 0 == versioncmp($operatingsystemmajrelease, '9')) {
```

Comment:
Same as above

Reply:
I updated the condition based on your suggestion. I needed additional guard for Fedora since its `osfamily` is `RedHat` and  `operatingsystemmajrelease` is `38` (or similar). python2-devel is still available on Fedora.

Reply:
On openEuler, `osfamily` is `Openeuler`.